### ISSUE TITLE
Plant Phenomics: tools not mentioned in in text

### ISSUE THREAD

**Issue by bedroesb** [elixir-europe/rdmkit#1315]
### Page: [Plant Phenomics](https://rdmkit.elixir-europe.org/plant_phenomics_assembly)

In pull request #1249, we are implementing changes to how we handle tools, resulting in different rules for rendering the tools table at the bottom of pages. Going forward, only tools referenced in the text will be included in the bottom table of the page. Some of the tools have been tagged with the page_id of this page, and we are currently exploring how to incorporate these tools into the text.

**Tools that are not mentioned in the text:**
- [x] **[Data INRAE](https://data.inrae.fr/)** - Dataverse for life sciences and agronomic related data
- [x] **[GnpIS](https://urgi.versailles.inrae.fr/gnpis/)** - A multispecies integrative information system dedicated to plant and fungi pests. It allows researchers to access genetic, phenotypic and genomic data. It is used by both large international projects and the French National Research Institute for Agriculture, Food and Environment.

Would you kindly review these tools and determine if they should be mentioned (as a link) in the text? If necessary, please propose changes to the text through a pull request on this temporary [instance of the RDMkit](https://bedroesb.github.io/rdmkit) by clicking on the pencil icon next to the title and following the instructions in this [guide](https://bedroesb.github.io/rdmkit/tool_resource_update). @afablondon, @cpommier, @bedroesb, @langeipk, @arendd, @DanFaria, @Isabelle-inrae, @proccaserra, @sebeier, @erlefloch

**Comment by bedroesb** [comment]
We will move to the new system in the week of the 24th of Juli, so please make sure changes are comments are made before this date. Thanks a lot in advance!

**Comment by bedroesb** [comment]
Since [Data INRAE](https://data.inrae.fr/) is deprecated and taken out of the tools and resources list, this issue can be closed.

**Comment by Isabelle-inrae** [comment]
Hi @Bert, data.inrae is not really obsolete, it has been migrated to recherche.data.gouv.fr and the url data.inrae.fr is redirected to the dedicated space (https://entrepot.recherche.data.gouv.fr/dataverse/inrae). 
I'm sorry I didn't react more quickly to this github issue because I wanted to check with the recherche.data.gouv.fr team what they recommended in terms of display (a data.inrae.fr with redirection or directly recherche.data.gouv.fr). Is it possible to leave the issue open a little longer?

**Comment by erlefloch** [comment]
Hi Isabelle, fyi with Anne-Françoise Adam-Blondon I replaced all Data INRAE occurences by the now official recherche.data.gouv.fr . I think using a redirection is not useful, let's just use the new one instead.

**Comment by Isabelle-inrae** [comment]
Hi Erwan, ok perfect, thanks for this information!

**Comment by bedroesb** [comment]
@erlefloch not a problem. I personally would not link towards a redirect. Cheers

**Comment by cpommier** [comment]
Hi @bedroesb  @erlefloch , it will in fact be a permanent redirect. it is the way to handle the INRAE section of the French national repository. 
Sincere appologies, but we should stick to data.inrae.fr

**Comment by erlefloch** [comment]
@cpommier what about CIRAD then ? It can also be about plant sciences. In my opinion it's not only about dead link but also granularity...

**Comment by cpommier** [comment]
You are right. Also, there will be a growing number of repositories in RDG, therefore sending people there and hoping fo the best might be problematic. Maybe a text like :   data.inrae.fr and CIRAD dataverse referenced in recherche.data.gouv ?
